music/demos/visualizer.ts

```diff
@@ -91,3 +101,243 @@ function startOrStop() {
     playBtn.textContent = 'Stop';
   }
 }
+
+
+// Section 2 ******
+
+const ADVANCED_MIDI_URL = './anna-magdalena-bach.mid';
```

Comment:
I wouldn't include a midi with this PR. We already have the option of loading you own midi, and i think that's enough

Reply:
I think it's important to have a medium complexity score in order to test and show the visualizer capabilities in the demo. If there's a problem about serving the MIDI, I would convert it to ProtoBuf in a separate .ts file and include it like Twinkle tune. It would help to handle manually some leaks in the MIDI conversor (like not handling the key signature of the MIDI file).